Project/zepeto-modules/Assets/Zepeto ScreenShot Module/ZepetoScript/SelfieCamera.ts

```diff
@@ -132,8 +127,9 @@ export default class SelfieCamera extends ZepetoScriptBehaviour {
         if (this.currentTarget == null || this.targetLookAt == null)
             return;
 
-        //this.CameraInput();
+        this.CameraInput();
```

Comment:
Why did you uncomment this?

Reply:
This function is related to camera rotation. It looks like it was previously disabled (commented out) due to an IK issue with selfie sticks. Since i've resolved that issue, re-enabled (uncommented) it.